substrate/client/rpc-spec-v2/src/common/events.rs

```diff
@@ -105,23 +105,41 @@ pub struct StorageResultErr {
 
 /// The result of a storage call.
 #[derive(Debug, Clone, PartialEq, Serialize, Deserialize)]
-#[serde(untagged)]
-pub enum ArchiveStorageResult {
+#[serde(rename_all = "camelCase")]
+#[serde(tag = "event")]
+pub enum ArchiveStorageEvent {
 	/// Query generated a result.
-	Ok(ArchiveStorageMethodOk),
+	StorageResult(StorageResult),
```

Comment:
I also note that `StorageResult` has no way to return `childTrieKey` unless I'm mistaken; should it be possible for this field to be returned?